build.gradle

```diff
@@ -32,16 +32,20 @@ repositories {
 }
 
 dependencies {
-    compileApi ('org.apache.jclouds:jclouds-core:2.0.1') 
-    compileApi ('com.google.auto.service:auto-service:1.0-rc3') 
+    compile ('com.google.code.gson:gson:2.5') {
+        force = true
```

Comment:
I force version of gson because jcloud don't support gson 2.8.0 (https://issues.apache.org/jira/browse/JCLOUDS-1160)